hasura.planx.uk/metadata/tables.yaml

```diff
@@ -40,6 +40,7 @@
           - id
           - input_errors
           - metadata
+          - node_id
```

Comment:
lots of formatting noise here - but this is the key change - there's a new column `node_id` that the public user has permission to `insert` !